core/templates/cards.html

```diff
@@ -261,6 +261,10 @@ <h4 class="modal-title">Face Off</h4>
                     card.removeAttr('onclick');
                     card.addClass('greyOut');
                 }
+                //Disables the onclick event for the card that is selected
+                else{
+                    card.removeAttr('onclick');
+                }
```

Comment:
hmm i see, so since it looks like we want to remove that onclick event regardless of whats happening in the if statements lets just move it out of them in geneneral so like:

if (gameMode != easyGame) {
                if (index != id || disableALL) {
                    card.addClass('greyOut');
                }
                  card.removeAttr('onclick');
            }